en2np/main.tra

```diff
@@ -1038,6 +1039,7 @@ emulation; यंत्रानुकरण
 enable; [इनेबल] v(अधिकार दिनु, प्राप्त गर्नु, सक्षम बनाउनु, लागु गर्नु, सुचारु गर्नु), ~on
 encoding; [इन्कोडिङ] सङ्केतन
 encourage; v(प्रोत्साहित)
+encouragement; n(प्राेत्साहन)
```

Comment:
you wrote
प्राे ‍→ प ‌+ ् + र ‌+ ा + े

but should be 
प्रो → प + ् + र ‌+ ो

try [m17n-ne](https://github.com/rhoit/m17n-ne) lipi for such auto correction